Jenkinsfile

```diff
@@ -1109,17 +1120,18 @@ pipeline {
                 stage('Functional on Leap 15') {
                     when {
                         beforeAgent true
-                        expression { ! skip_stage('func-test-leap15') }
+                        allOf {
+                            expression { ! skip_stage('func-test') }
+                            expression { ! skip_stage('func-test-leap15') }
+                        }
                     }
                     agent {
                         label 'ci_vm9'
                     }
                     steps {
-                        functionalTest target: 'leap15',
-                                       inst_repos: daos_repos('leap15'),
-                                       inst_rpms: get_daos_packages('leap15') + ' ' +
-                                                  functional_rpms('leap15') + ' ' +
-                                                  qb_inst_rpms_functional()
+                        functionalTest inst_repos: daos_repos(),
+                                       inst_rpms: get_daos_packages() +
+                                                  ' ' + functional_packages()
```

Comment:
Why does this VM stage need `get_daos_packages()` when the hardware stages below don't?

This applies to the EL7 VM stage above also.
￼

Reply:
Missed changing those two stages.   The functional_packages() function now also does the get_daos_packages() internal to it.